src/main/java/frc/robot/Constants.java

```diff
@@ -16,7 +18,18 @@ public class Drive {
         public static final double WHEEL_DIAMETER_METERS = Units.inchesToMeters(4);
 
     }
-
+    public class Arm{
+        public static final int ENCODER_COUNT = 8192;
+        public static final int MOTOR_1_ID = 0;
```

Comment:
let's standardize naming to `ARM_MOTOR_ONE` / `TWO`

Reply:
I am initially assigning ID's 11 and 12 to arm. Please change these

1-8 swerve
9-10 undertaker
11-12 arm
13-16 shooter

Reply:
The swerve actually needs CAN IDs 1-12, so add 4 to your subsystem can IDs.